packages/workflows/src/cli/create/Create.common.options.ts

```diff
@@ -97,7 +97,8 @@ export const CreateCommonOptions: { [key: string]: ICommandOptionDefinition } =
         name: "variables",
         aliases: ["vs"],
         description: "Includes a list of variables for the workflow. " +
-        "The variables that you specify here take precedence over the variables that are specified in the workflow variable input file.",
+        "The variables that you specify here take precedence over the variables that are specified in the workflow variable input file." +
```

Comment:
Looks like there is a missing space here: `file.Make`. Also, missing period at the end of the sentence.